enterprise/models.py

```diff
@@ -764,3 +774,37 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+
+class EnterpriseIntegratedChannel(TimeStampedModel):
+    """
+    EnterpriseIntegratedChannel is an external system that the Enterprise Customer uses to manage enterprise
+    related data. Each of these requires an edx plugin in order to send data to these systems, this model stores
+    information related to what and where to send data from the enterprise app.
+    """
+    id = models.PositiveIntegerField(primary_key=True)
+    name = models.CharField(max_length=255, blank=False, null=False, help_text=_("Third Party name."))
+    data_type = models.CharField(max_length=100, blank=False, null=False, help_text=_("Data Type"))
+    plugin_endpoint = models.CharField(max_length=255, blank=False, null=False, help_text=_("Internal Plugin endpoint."))
```

Comment:
I wanted to make data_type an enum, it's supposed to be something like 'course' or 'learner', which are the two types of data so far that we are concerned with sending to an external system. I feel like there is probably a better way to express plugin_endpoint, which is meant to represent our internal endpoint for forwarding data to a particular channel's plugin. It's not clear yet if these will just be function calls or be made through api calls, so I'm not sure what this should look like yet.

The idea is you could configure multiple channels to receive data for a given enterprise, and all of the channels that are of the same data type would have plugins that follow a common interface for that data type and deal with the details of sending to a specific channel.